posthog/queries/test/test_trends.py

```diff
@@ -1576,17 +1576,24 @@ def test_trends_regression_filtering_by_action_with_person_properties(self):
                 properties=[{"key": "bar", "type": "person", "value": "a", "operator": "icontains"}],
             )
             event_filtering_action.calculate_events()
```

Comment:
It looks like this test was already passing beforehand with filtering be person property, is the change here just to make sure that trends also works without a person filter?

Reply:
The added lines crashed w/o app code change because they didn't trigger a join. Rather than a new test, this test seemed close enough to be what we needed, hence editing it.